### PR TITLE
Capacitor & PWA Support

### PR DESCRIPTION
**Pull request by tlancina**
This branch is currently up as a PWA at https://ionifits.now.sh

### CODE REVIEW

File: ionic.config.json [hunk 1]
```diff
@@ -1,7 +1,16 @@
 {
   "name": "Ionifits",
   "integrations": {
-    "cordova": {}
+    "capacitor": {},
+    "enterprise": {
+      "keyId": 45,
+      "productKey": "key_11375d3f0a104b27a3e5d92b442245ef",
```
Comment (dotNetkow) [inline ionic.config.json:7]: see DM

File: src/app/services/authentication.service.ts [hunk 2]
```diff
@@ -30,24 +30,26 @@ const auth0CordovaConfig : IonicAuthOptions = {
...
   authConfig: 'auth0',
   // The platform which we are running on
   platform: 'web',
   // client or application id for provider
-  clientID: 'ihSRqLLa2z33PTyeNNlI2uxgsqorb08l',
+  clientID: 'vbOi3NDg3eRfQTUL4FuTqhaQHcHKPgOh',
   // the discovery url for the provider
   // OpenID configuration
-  discoveryUrl: 'https://ionicorg.auth0.com/.well-known/openid-configuration',
+  discoveryUrl: 'https://ionicteam.auth0.com/.well-known/openid-configuration',
```
Comment (dotNetkow) [inline src/app/services/authentication.service.ts:42]: is this the same org? I set up a special user account for Auth0 sign-in - would want to ensure that everything's configured correctly for web and mobile. yea?

File: src/app/services/expense.service.ts [hunk 3]
```diff
@@ -31,59 +28,31 @@ export class ExpenseService {
...
-      encodingType: this.camera.EncodingType.JPEG,
-      mediaType: this.camera.MediaType.PICTURE
-    }
-
-    const imageData = await this.camera.getPicture(options);
-    const resolvedImg = this.webview.convertFileSrc(imageData);
-    return {
-      sanitizedReceiptImage: this.sanitizer.bypassSecurityTrustUrl(resolvedImg),
-      originalImage: imageData
+      resultType: CameraResultType.Base64
```
Comment (dotNetkow) [inline src/app/services/expense.service.ts:31]: wiped out my beautiful CRUD stuff! haha.  Can we merge this PR after we get all platforms working? Don't want to break ios/android... I think it wouldn't be too difficult, actually. I implemented almost the same functionality in the [First App tutorial](https://ionicframework.com/docs/angular/your-first-app).

File: src/app/services/identity.service.ts [hunk 4]
```diff
@@ -15,7 +15,7 @@ export class IdentityService extends IonicIdentityVaultUser<DefaultSession> {
 
   constructor(public platform: Platform, private browserAuthPlugin: BrowserAuthPlugin) {
     super(platform, {
-      authMode: AuthMode.BiometricOnly,
+      // authMode: AuthMode.BiometricOnly,
```
Comment (dotNetkow) [inline src/app/services/identity.service.ts:18]: Tested this on iOS before - it worked with my Face Id. Reason to take it out? on Android, it simply didn't do anything.

Comment (tlancina) [reply src/app/services/identity.service.ts:18]: See https://github.com/ionic-team/ionifits/pull/22/commits/7faa13ad0a50431152afa682293fab3c5cb4f942

File: src/app/services/expense.service.ts (hunk 3 above)
Comment (tlancina) [reply src/app/services/expense.service.ts:31]: Yeah, it's not really meant to be merged in right now.  Want me to change to WIP?

File: src/app/services/authentication.service.ts (hunk 2 above)
Comment (tlancina) [reply src/app/services/authentication.service.ts:42]: I don't think I have access to `ionicorg` on Auth0, so I set up a new app in the Ionic team.

**Pull request merged**